utils/src/server/rpc_module.rs

```diff
@@ -489,6 +549,7 @@ impl SubscriptionSink {
 		serde_json::to_string(&JsonRpcNotification {
 			jsonrpc: TwoPointZero,
 			method: self.method,
+			// TODO: `params` is a bit weird here. It's a result payload not really a set of params?
```

Comment:
This is basically the same as `jsonrpc crate` and you could not send it in another field without diverging for  spec but at little bit weird yes.

I think you could create an issue for this but should most likely be kept out-of this PR

Reply:
https://www.jsonrpc.org/specification#notification

Reply:
I guess the weirdness comes from us using `JsonRpcNotification` as a response (subscription ––> subscriber). In the spec, notifications are sent from the client ––> server right?

Reply:
> I think you could create an issue for this but should most likely be kept out-of this PR

Definitely.

Reply:
The server is sending subscription responses as "notifications", I don't really know what else to do without introduces stuff outside the jsonrpc spec.